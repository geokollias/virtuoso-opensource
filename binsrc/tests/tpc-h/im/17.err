Starting Impala Shell without Kerberos authentication
Connected to ip-10-0-1-251.ec2.internal:21000
Server version: impalad version 2.1.3-cdh5 RELEASE (build 20816e26a150d20d2c92c470aa40b342521b243e)
Query: use TPCH
Query: select sum(l_extendedprice) / 7.0 as avg_yearly
from
	lineitem,
	part
where
	p_partkey = l_partkey
	and p_brand = 'Brand#23'
	and p_container = 'MED BOX'
	and l_quantity < (
		select
			2e-1 * avg(l_quantity)
		from
			lineitem
		where
			l_partkey = p_partkey
	)
Fetched 1 row(s) in 476.29s
