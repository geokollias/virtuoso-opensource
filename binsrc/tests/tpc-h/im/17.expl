[?1034h+------------------------------------------------------------------------------------+
| Explain String                                                                     |
+------------------------------------------------------------------------------------+
| Estimated Per-Host Requirements: Memory=860.03MB VCores=3                          |
| WARNING: The following tables are missing relevant table and/or column statistics. |
| tpch.part                                                                          |
|                                                                                    |
| 12:AGGREGATE [FINALIZE]                                                            |
| |  output: sum:merge(l_extendedprice)                                              |
| |  hosts=1 per-host-mem=unavailable                                                |
| |  tuple-ids=6 row-size=8B cardinality=1                                           |
| |                                                                                  |
| 11:EXCHANGE [UNPARTITIONED]                                                        |
| |  hosts=1 per-host-mem=unavailable                                                |
| |  tuple-ids=6 row-size=8B cardinality=1                                           |
| |                                                                                  |
| 06:AGGREGATE                                                                       |
| |  output: sum(l_extendedprice)                                                    |
| |  hosts=1 per-host-mem=10.00MB                                                    |
| |  tuple-ids=6 row-size=8B cardinality=1                                           |
| |                                                                                  |
| 05:HASH JOIN [LEFT SEMI JOIN, BROADCAST]                                           |
| |  hash predicates: p_partkey = l_partkey                                          |
| |  other join predicates: l_quantity < 0.2 * avg(l_quantity)                       |
| |  hosts=1 per-host-mem=295.50MB                                                   |
| |  tuple-ids=0,1 row-size=44B cardinality=600037902                                |
| |                                                                                  |
| |--10:EXCHANGE [BROADCAST]                                                         |
| |  |  hosts=1 per-host-mem=0B                                                      |
| |  |  tuple-ids=4 row-size=12B cardinality=23473626                                |
| |  |                                                                               |
| |  09:AGGREGATE [FINALIZE]                                                         |
| |  |  output: avg:merge(l_quantity)                                                |
| |  |  group by: l_partkey                                                          |
| |  |  hosts=1 per-host-mem=295.50MB                                                |
| |  |  tuple-ids=4 row-size=12B cardinality=23473626                                |
| |  |                                                                               |
| |  08:EXCHANGE [HASH(l_partkey)]                                                   |
| |  |  hosts=1 per-host-mem=0B                                                      |
| |  |  tuple-ids=3 row-size=12B cardinality=23473626                                |
| |  |                                                                               |
| |  03:AGGREGATE                                                                    |
| |  |  output: avg(l_quantity)                                                      |
| |  |  group by: l_partkey                                                          |
| |  |  hosts=1 per-host-mem=295.50MB                                                |
| |  |  tuple-ids=3 row-size=12B cardinality=23473626                                |
| |  |                                                                               |
| |  02:SCAN HDFS [tpch.lineitem, RANDOM]                                            |
| |     partitions=1/1 files=82 size=20.20GB                                         |
| |     table stats: 600037902 rows total                                            |
| |     column stats: all                                                            |
| |     hosts=1 per-host-mem=176.00MB                                                |
| |     tuple-ids=2 row-size=12B cardinality=600037902                               |
| |                                                                                  |
| 04:HASH JOIN [INNER JOIN, BROADCAST]                                               |
| |  hash predicates: l_partkey = p_partkey                                          |
| |  hosts=1 per-host-mem=5.04MB                                                     |
| |  tuple-ids=0,1 row-size=44B cardinality=600037902                                |
| |                                                                                  |
| |--07:EXCHANGE [BROADCAST]                                                         |
| |  |  hosts=1 per-host-mem=0B                                                      |
| |  |  tuple-ids=1 row-size=24B cardinality=200000                                  |
| |  |                                                                               |
| |  01:SCAN HDFS [tpch.part, RANDOM]                                                |
| |     partitions=1/1 files=3 size=648.75MB                                         |
| |     predicates: p_brand = 'Brand#23', p_container = 'MED BOX'                    |
| |     table stats: 20000000 rows total                                             |
| |     columns missing stats: p_brand, p_container                                  |
| |     hosts=1 per-host-mem=264.00MB                                                |
| |     tuple-ids=1 row-size=24B cardinality=200000                                  |
| |                                                                                  |
| 00:SCAN HDFS [tpch.lineitem, RANDOM]                                               |
|    partitions=1/1 files=82 size=20.20GB                                            |
|    table stats: 600037902 rows total                                               |
|    column stats: all                                                               |
|    hosts=1 per-host-mem=264.00MB                                                   |
|    tuple-ids=0 row-size=20B cardinality=600037902                                  |
+------------------------------------------------------------------------------------+
