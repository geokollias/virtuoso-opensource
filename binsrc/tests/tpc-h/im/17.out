[?1034h+------------+
| avg_yearly |
+------------+
| NULL       |
+------------+
